tfx/components/example_gen/utils.py

```diff
@@ -276,19 +322,40 @@ def _retrieve_latest_span_version(
   logging.info('Regex pattern for split %s: %s', split.name,
                split_regex_pattern)
 
+  latest_span_elems = None
+  latest_span_int = None
+  latest_version = None
+
   files = tf.io.gfile.glob(split_glob_pattern)
 
   for file_path in files:
     result = re.search(split_regex_pattern, file_path)
     if result is None:
       raise ValueError('Glob pattern does not match regex pattern')
 
-    span_str = result.group(SPAN_PROPERTY_NAME)
-    try:
-      span_int = int(span_str)
-    except ValueError:
-      raise ValueError('Cannot find %s number from %s based on %s' %
-                       (SPAN_PROPERTY_NAME, file_path, split_regex_pattern))
+    span_strs = None
```

Comment:
nit, s/span_strs/matched_span_tokens

Reply:
updated. also changed `latest_span_elems` to `latest_span_tokens`